resources/views/platform/components/game/current-progress/big-status-label.blade.php

```diff
@@ -35,15 +37,35 @@
     $statusLabel = "Awarded";
 }
 
+$isBeaten = $isBeatable && ($isBeatenHardcore || $isBeatenSoftcore);
+
 // This case can occur on legacy completion/mastery awards
 // where progression achievements were added after the user
 // had already mastered the game. It's an edge case, but we
 // try to gracefully handle it anyway.
-if (($isCompleted || $isMastered) && ($isBeatable && !$isBeatenHardcore && !$isBeatenSoftcore)) {
+if (($isCompleted || $isMastered) && !$isBeaten) {
     $statusLabel = "Unbeaten";
 }
 ?>
 
-<p class="text-lg {{ $colorClassName }} mb-2.5 mt-0.5">
-    {{ $statusLabel }}
-</p>
\ No newline at end of file
+<div class="text-lg {{ $colorClassName }} mb-1.5 mt-0.5 flex items-center gap-x-1">
+    <p>{{ $statusLabel }}</p>
+
+    @hasfeature('beat')
+        @if (!$isBeaten)
```

Comment:
Should this also be tied to `$isBeatable`?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/a0e5bb09-319d-4663-a365-205451710c27)

Reply:
Yep, resolved in https://github.com/RetroAchievements/RAWeb/pull/1859/commits/bfbca79c9b38b3e3947c8f6a41d20e43b580fb58.